<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudent</class>
 <widget class="QWidget" name="AddStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="head">
      <property name="text">
       <string>Добавление студента</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="lNameText">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLineEdit" name="lName"/>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="fNameText">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLineEdit" name="fName"/>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="mNameText">
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLineEdit" name="mName"/>
      </item>
      <item>
       <widget class="QLabel" name="numberGroup">
        <property name="text">
         <string>Номер группы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numGroup"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="save">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
